StRoot/StFstSimMaker/StFstFastSimMaker.cxx

```diff
@@ -144,6 +156,10 @@ void StFstFastSimMaker::FillSilicon(StEvent *event) {
 
 	float X0[] = {0, 0, 0, 0, 0, 0};
 	float Y0[] = {0, 0, 0, 0, 0, 0};
+
+	if ( mGEANTPassthrough ){
+		LOG_INFO << "FST Hits using GEANT xyz directly (no raster etc.)" << endm;
+	}
 	
 	if (mRaster > 0)
```

Comment:
Is there still a need to randomize the X,Y center of each plane?

Reply:
I am not sure what you mean? Maybe you mean blur the position of the hits according to a resolution - thats not a bad idea. I can add that, but please let me know if you meant something different

Reply:
It looks like I commented on the wrong line of code.  Just so that we're on the same page, I'm referring to lines 157 and 158, and the if ( mRaster>0 ) block on line 164.  

This code appears suspect to me.  It doesn't smear the hit positions.  It systematically shifts the (x,y) center of each disk to the vertices of a hexagon of 'radius' mRaster.  As I recall, the idea was to provide a simple way to introduce a small stereo angle between the different planes, to improve tracking resolution during design studies.  The class defaults to mRaster=0, so... it probably hasn't caused a problem.  But it looks to me to be dead code that no longer serves a purpose, and thus should be cleaned up.  Time being at a premium... at a minimum I would eliminate any setter methods.

Reply:
Thanks - this is taken care of in d8801c1. I removed the options to set Raster and removed the additional logic in the main code.

Reply:
@klendathu2k if you are good with this can you approve?